eng/targetingpacks.targets

```diff
@@ -164,7 +164,9 @@
           AfterTargets="ResolveTargetingPackAssets">
     <ItemGroup>
       <RuntimeFramework Version="$(ProductVersion)"
-                        Condition="'%(RuntimeFramework.FrameworkName)' == '$(LocalFrameworkOverrideName)'" />
+                        Condition="'$(StageTwoBuild)' != 'true' and '%(RuntimeFramework.FrameworkName)' == '$(LocalFrameworkOverrideName)'" />
+      <RuntimeFramework Version="$(BundledNETCoreAppPackageVersion)"
+                        Condition="'$(StageTwoBuild)' == 'true' and '%(RuntimeFramework.FrameworkName)' == '$(LocalFrameworkOverrideName)'" />
```

Comment:
Apply the condition directly on the Version attribute so that you don't need two items:

```
<RuntimeFramework Condition="'%(RuntimeFramework.FrameworkName)' == '$(LocalFrameworkOverrideName)'">
  <Version Condition="'$(StageTwoBuild)' != 'true'">$(ProductVersion)</Version>
  <Version Condition="'$(StageTwoBuild)' == 'true'">$(BundledNETCoreAppPackageVersion)</Version>
</RuntimeFramework>
```